qemu/tests/cfg/qemu_disk_img.cfg

```diff
@@ -60,7 +60,12 @@
                     variants:
                         - @default:
                         - compressed:
-                            convert_compressed = yes
+                            variants:
+                                - zlib:
+                                    convert_compressed = yes
```

Comment:
Thanks Yongxue's inputs here, I guessed Tingting wanna a way to set the default compression type instead of a specific one

Reply:
> Thanks Yongxue's inputs here, I guessed Tingting wanna a way to set the default compression type by qemu-img itself instead of a specific one

Hi @hellohellenmao 
if your purpose is to aim at the default compression, I suggest that the variant name is `@default:`, since the `- zlib:` is a bit of hardcode that we can not make sure the default will be updated in the qemu product in the future. 
Shortly, we suggest that the variant naming is better to be close to your intention.
Thanks.
CC @zhencliu